test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/CompleteTagHelperDescriptorComparer.cs

```diff
@@ -23,16 +23,22 @@ bool IEqualityComparer<TagHelperDescriptor>.Equals(TagHelperDescriptor descripto
             return base.Equals(descriptorX, descriptorY) &&
                    // Tests should be exact casing
```

Comment:
Normal comparer does `OrdinalIgnoreCase` comparison, for tests we want `Ordinal`